modules/rostests/kmtests/ntos_cc/CcMapData_drv.c

```diff
@@ -300,16 +311,13 @@ PerformTest(
                             PKTHREAD ThreadHandle;
 
 #ifdef _X86_
-                            /* FIXME: Should be fixed, will fail under certains conditions */
-                            ok(TestContext->Buffer > (PVOID)0xC1000000 && TestContext->Buffer < (PVOID)0xDCFFFFFF,
-                               "Buffer %p not mapped in system space\n", TestContext->Buffer);
-#else
-#ifdef _M_AMD64
-                            ok(TestContext->Buffer > (PVOID)0xFFFFF98000000000 && TestContext->Buffer < (PVOID)0xFFFFFA8000000000,
-                               "Buffer %p not mapped in system space\n", TestContext->Buffer);
+                            ok(TestContext->Buffer >= MI_SYSTEM_CACHE_START && TestContext->Buffer < MI_PAGED_POOL_START,
```

Comment:
This is wrong.
See Windows Internals 4th edition, Table 11-1, page 664.

Reply:
Thanks for the pointer! `0xDCFFFFFF` is for 3GB+TS, which does not seem to be our typical/current case(s).
What is wrong exactly? `MI_PAGED_POOL_START` is for standard 2GB, `LargeSystemCache=1` does not seem to be our typical/current case(s) either.
From my point of view, this is an improvement/sync with regard to current state (ReactOS and Test WHS).

Should the FIXME remain? Maybe with a better explanation about the 3 cases?